torch/_torch_docs.py

```diff
@@ -7385,6 +7385,45 @@ def merge_dicts(*dicts):
             [3, 2, 1, 0]])
 """.format(**common_args))
 
+add_docstr(torch.msort,
+           r"""
+msort(input, descending=False, *, out=None) -> (Tensor, LongTensor)
+
+Sorts the elements of the :attr:`input` tensor along the first dimension
+in ascending order by value.
+
+If :attr:`descending` is ``True`` then the elements are sorted in descending
+order by value.
+
+A namedtuple of (values, indices) is returned, where the `values` are the
+sorted values and `indices` are the indices of the elements in the original
+`input` tensor.
+
+.. note:: `torch.msort(a)` is equivalent to `torch.sort(a, dim=0)`.
+          See also :func:`torch.sort`.
+
+Args:
+    {input}
+    descending (bool, optional): controls the sorting order (ascending or descending)
+
+Keyword args:
+    out (tuple, optional): the output tuple of (`Tensor`, `LongTensor`) that can
+        be optionally given to be used as output buffers
+
+Example::
+
+    >>> x = torch.randn(3, 4)
```

Comment:
When a name is needed for only a single tensor use "t". Print the matrix before sorting it so users can understand the before and after of sorting.

Reply:
Addressed!